CI: Is scipy.scipy Windows Python36-32bit-full working?

Is the scipy.scipy Windows Python36-32bit-full testing working?
I'm seeing a few PRs (gh-9723, gh-9720, gh-9719)  whose only failing test is this one Azure pipeline build.
The log for gh-9723 (https://dev.azure.com/scipy-org/SciPy/_build/results?buildId=315) shows several error messages, but it's not clear which is the first fatal error.

E.g.
```
C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\VC\Tools\MSVC\14.16.27023\bin\HostX86\x86\link.exe /nologo /INCREMENTAL:NO /LTCG /MANIFEST:EMBED,ID=1 /LIBPATH:C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\VC\Tools\MSVC\14.16.27023\ATLMFC\lib\x86 /LIBPATH:C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\VC\Tools\MSVC\14.16.27023\lib\x86 /LIBPATH:C:\Program Files (x86)\Windows Kits\NETFXSDK\4.6.1\lib\um\x86 /LIBPATH:C:\Program Files (x86)\Windows Kits\10\lib\10.0.17763.0\ucrt\x86 /LIBPATH:C:\Program Files (x86)\Windows Kits\10\lib\10.0.17763.0\um\x86 /LIBPATH:C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\LIB /LIBPATH:C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\ATLMFC\LIB /LIBPATH:C:\Program Files (x86)\Windows Kits\10\lib\10.0.17763.0\ucrt\x86 /LIBPATH:C:\Program Files (x86)\Windows Kits\NETFXSDK\4.6.1\lib\um\x86 /LIBPATH:C:\Program Files (x86)\Windows Kits\10\lib\10.0.17763.0\um\x86 _configtest.obj /OUT:_configtest.exe
2019-01-27T22:28:00.4248485Z   _configtest.obj : error LNK2001: unresolved external symbol _open_memstream
2019-01-27T22:28:00.4249043Z   _configtest.exe : fatal error LNK1120: 1 unresolved externals
2019-01-27T22:28:00.4249228Z   failure.

error: Command "C:\ProgramData\Chocolatey\bin\gfortran.exe -Wall -g -ffixed-form -fno-second-underscore -O3 -funroll-loops -Ic:\hostedtoolcache\windows\python\3.6.4\x86\lib\site-packages\numpy\core\include -c -c scipy\fftpack\src\dfftpack\dcosqb.f -o build\temp.win32-3.6\scipy\fftpack\src\dfftpack\dcosqb.o" failed with exit status 1
2019-01-27T22:28:01.6594577Z   Building wheel for scipy (setup.py): finished with status 'error'
2019-01-27T22:28:01.6596596Z   Failed building wheel for scipy
2019-01-27T22:28:01.6600726Z   Running setup.py clean for scipy
2019-01-27T22:28:02.0478317Z   `setup.py clean` is not supported, use one of the following instead:
2019-01-27T22:28:02.0694984Z   Failed cleaning build dir for scipy
2019-01-27T22:28:02.0702417Z Failed to build scipy
2019-01-27T22:28:03.3313460Z ERROR: Failed to build one or more wheels
```

Is this particular build/test reliable currently?

--- a/azure-pipelines.yml
+++ b/azure-pipelines.yml
@@ -86,7 +86,7 @@ jobs:
   - powershell: |
       # NOTE: can probably (eventually) abstract this 
       # upstream in Microsoft repo to support x86 natively
-      choco install -y mingw --forcex86 --force
+      choco install -y mingw --forcex86 --force --version=5.3.0
     displayName: 'Install 32-bit mingw for 32-bit builds'
     condition: eq(variables['BITS'], 32)
   - script: python -m pip install numpy cython==0.28.5 pytest pytest-timeout pytest-xdist pytest-env pytest-faulthandler Pillow mpmath matplotlib

